### PR TITLE
Fix the meaning of few sentences in set_1.md

### PR DESCRIPTION
**Pull request by brgirgis**
Fix the meaning of few sentences.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@brgirgis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/windows-commands/set_1.md [hunk 1]
```diff
@@ -80,15 +80,15 @@ set /a <variable>=<expression>
 
 ## Examples
 
-To set an environment variable named *TEST^1*, type:
+To set an environment variable named `testVar` the value of *TEST^1*, type:
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/set_1.md:83]: improvement suggestion for a better English sentence structure:
```suggestion
To set the value of *TEST^1* for the environment variable named `testVar`, type:
```

File: WindowsServerDocs/administration/windows-commands/set_1.md [hunk 2]
```diff
@@ -80,15 +80,15 @@ set /a <variable>=<expression>
...
 
 ```
 set testVar=test^^1
 ```
 
 The **set** command assigns everything that follows the equal sign (=) to the value of the variable. Therefore, if you type `set testVar=test^1`, you'll get the following result, `testVar=test^1`.
 
-To set an environment variable named *TEST&1*, type:
+To set an environment variable the value *TEST&1*, type:
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/set_1.md:91]:
```suggestion
To set the value *TEST&1* for the environment variable `testVar`, type:
```

**Comment by illfated** [comment]
Good idea to clarify the sentences. I just added improvement suggestions for them. Hope you find them useful. Please commit the suggestion if you agree.

**Comment by PRMerger15** [comment]
@eross-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by eross-msft — APPROVED** [review]

**Comment by brgirgis** [comment]
If you let one person steal everyone's errata finds and author it for themselves, no one will ever contribute to repo twice! Enjoy your fake glory @illfated 😉

**Comment by eross-msft** [comment]
@brgirgis @illfated  To be clear, you both get credit for updating the article. Illfated's version was just more grammatically correct.

**Comment by illfated** [comment]
Thank you. I was just about to write a comment asking when contributing with added improvements became stealing, I digress. My goal is to help others achieve correct Microsoft codestyle & phrasing, so the final result will be merged. Otherwise, it could take longer before the suggested changes get accepted and merged. I hope this answers your questions about my motives for contributing.

**Comment by brgirgis** [comment]
If you really care, go read the docs. Spend days and hours reviewing and find your own errata to correct. Basically don't ride on someone's efforts. Beside, one should always try to fix with minimal changes to original author's version. Which I did in my change. From my point of view, @illfated , you are just a scavenger.

**Comment by eross-msft** [comment]
@brgirgis We appreciate all contributions regardless of when or where they come from. You are both valued members of the contributor community and as such appear as contributors to the article. At this point, you are veering into code of conduct territory. Please refrain from attacking other contributors. Thank you!

**Comment by brgirgis** [comment]
Right, last time to fix anything for you. Enjoy your "code of conduct"!

**Pull request merged**